README.md

```diff
@@ -10,6 +10,7 @@ If you would like to be notified of future conferences that would be added to th
 
 | Conference | Date | Venue | Description |
 |------------|------|-------|-------------|
+| [Great Indian Developer Summit](http://www.developermarch.com/developersummit/index.html) | 24-27 Apr | Bengaluru | 	With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem.
```

Comment:
Can you change the chronological order here? Like you mentioned, it was wrong before.

If not, I am happy to merge the PR as-is and will fix this in a followup. :)